torch/fx/experimental/proxy_tensor.py

```diff
@@ -434,24 +442,10 @@ def inner_torch_dispatch(self, func, types, args=(), kwargs=None):
         if not self.enable_tracing:
             return func(*args, **kwargs)
 
-        if symbolic_shapes.is_symbolic_op(func):
-            with self.restore():
-                return symbolic_shapes.handle_symbolic_op(func, args, kwargs)
-
         if func in [prim.device.default]:
             return func(*args, **kwargs)
 
-        out = proxy_call(self, func, args, kwargs)
-
-        def assert_proxy_tensor(e):
```

Comment:
i think this is ok to remove.  if we want to keep it alive in some form, does it seem any better to do the assertion as a validation pass on the final graph?

Reply:
Hmm, if we have a mode where no constants are allowed (user use of `make_fx` can refer to constants, so you cannot distinguish these cases), this validation may be possible.